examples/serenity/voice/src/main.rs

```diff
@@ -63,14 +63,15 @@ async fn main() {
         .await
         .expect("Err creating client");
 
-    let _ = client.start().await.map_err(|why| println!("Client ended: {:?}", why));
+    if let Err(why) = client.start().await {
+        println!("Err starting client: {:?}", why))
```

Comment:
Aside from Acdenis's comments, you've put in an extra bracket here. The example will not compile as-is.

Reply:
yes i should learn to use vscode for prs sorry